subprojects/emcee/k8s/emcee.yaml

```diff
@@ -0,0 +1,44 @@
+---
```

Comment:
I don't remember all details but could you add what we try to do via this yaml file
Could yaml have a comment?
As I see we create a `deployment` related to `service`. But do we need a `service`? Or we could do the same without `service` only with `deployment`?

Reply:
Add comments